test-suite/non-functional-tests/test-scenario-1/start-docker-test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+openssl pkcs12 -export -out jmeterkeystore.p12 -inkey ../certs/client.key -in ../certs/client.crt -passout pass:password
```

Comment:
Is this a hardcoded passphrase?

Reply:
I'm _guessing_ there is some technical need to password protect the certificate? Seems annoying.